src/admin/containers/AddonPage/index.js

```diff
@@ -132,7 +132,7 @@ function mapStateToProps(state, ownProps) {
 const CurrentAddonPage = asyncConnect([{
```

Comment:
I know it's not really related to this patch but it'd be nice to see `compose` here as we use it elsewhere and it makes things a bit easier to read 😄